src/api/vade_evan_api.rs

```diff
@@ -70,6 +71,49 @@ impl VadeEvan {
         }
     }
 
+    /// Creates a credential request. This function is used to create a credential request which is sent to Issuer
+    ///
+    /// # Arguments
+    ///
+    /// * `issuer_public_key` - issuer public key
+    /// * `bbs_secret` - master secret of the holder/receiver
+    /// * `credential_values` - JSON string with cleartext values to be signed in the credential
+    /// * `credential_offer` - JSON string with credential offer by issuer
+    /// * `credential_schema` - JSON string with credential schema
```

Comment:
Was not in the ticket, but if we could resolve the schema from a DID, this would make the API a bit easier to use. The offer PR has code to resolve it, that we could re-use here. ^^